### ISSUE TITLE
Syncing with spacemacs-layouts layer

### ISSUE THREAD

**Issue by jupl** [syl20bnr/spacemacs#5573]
#### Description
When I removed the `eyebrowse` layer due to `spacemacs-layouts` layer and I sync configuration an error appears.

#### Reproduction guide
- Start Emacs
- Press `SPC f e R`

*Observed behaviour:*
`*Warnings*` buffer appears with the following:
```
Error (use-package): eyebrowse :init: Key sequence l w starts with non-prefix key l
```

*Expected behaviour:*
Not observed behavior.

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.14
- Spacemacs branch: develop (rev. 8266b5f)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((clojure :variables clojure-enable-fancify-symbols t)
 typescript clojure colors dockerfile emacs-lisp html github markdown osx react restclient spacemacs-layouts swift syntax-checking vimscript yaml
 (auto-completion :variables auto-completion-enable-help-tooltip t)
 (git :variables git-magit-status-fullscreen t magit-push-always-verify nil)
 (javascript :variables js2-mode-show-parse-errors nil js2-mode-show-strict-warnings nil)
 (org :variables org-src-preserve-indentation t org-startup-folded 'showall)
 (shell :variables shell-default-term-shell "/bin/zsh" shell-file-name "/bin/sh")
 (version-control :variables version-control-diff-tool 'diff-hl))
```

**Comment by syl20bnr** [comment]
Should be fixed in develop.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
